src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1765,9 +1781,9 @@ private static string GetLinuxParentPath(string path)
 
         private static Dictionary<string, string> DelimitedTextToDictionary(string text, string fieldDelimiter = "=", string rowDelimiter = "\n")
         {
-            return text.Split(rowDelimiter)
-                .Select(line => { var parts = line.Split(fieldDelimiter); return new KeyValuePair<string, string>(parts[0], parts[1]); })
-                .ToDictionary(kv => kv.Key, kv => kv.Value);
+            return text.Trim().Split(rowDelimiter)
+                .Select(line => { var parts = line.Trim().Split(fieldDelimiter); return new KeyValuePair<string, string>(parts[0], parts[1]); })
```

Comment:
Your code for this looked cleaner (also the one I had in the chat) - it was not creating KeyValuePair objects. Was there a reason not to use it?